thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -768,6 +770,7 @@ public Response replayAnomalyFunctionByFunctionId(@PathParam("functionId") long
     DateTime replayStart = ISODateTimeFormat.dateTimeParser().parseDateTime(replayStartTimeIso);
     DateTime replayEnd = ISODateTimeFormat.dateTimeParser().parseDateTime(replayEndTimeIso);
     FunctionReplayRunnable functionReplayRunnable = new FunctionReplayRunnable(detectionJobScheduler, anomalyFunctionDAO, mergedAnomalyResultDAO, rawAnomalyResultDAO, new HashMap<String, String>(), id, replayStart, replayEnd, false);
+    functionReplayRunnable.setForceBackfill(false);
```

Comment:
Thanks for your suggestions @bl44 and @cyenjung. I'll close this pr and create a new one to generate initial anomalies report. - i.e. That's our actual goal. :)